pyproject.toml

```
@@ -31,7 +31,7 @@ dependencies = [
   "importlib-metadata~=6.0;python_version<'3.10'",
   "importlib-resources~=5.0;python_version<'3.9'",
   "nbclient~=0.5.10",
-  "nbdime",
+  "nbdime<5,>=4",
```

```suggestion
  "nbdime>=4",
```

actually I think remove the upper-pinning, because otherwise if/when v5 comes out, pip could start pulling in older versions of this package (since they have no pinning)

---

I will be happy to follow your suggestion.

However, may I ask if that isn't a good thing for stability reasons that only `pytest-notebook` has the authority to define compatible versions of dependencies? On the other hand, I know that is usual practice not to introduce too many pinnings on software which is actually a library, and rather make it compatible with different versions of dependency packages, in order not to trip dependent software too much. `pytest-notebook` may qualify as such a package, right?

---

its always a tricky subject, but for now I think I'll stick with no pinning and look at that later